ntbs-integration-tests/Helpers/TestDocumentExtensions.cs

```diff
@@ -1,11 +1,19 @@
 using AngleSharp.Html.Dom;
 using ntbs_service.Helpers;
+using Xunit;
 
 namespace ntbs_integration_tests.Helpers
 {
     public static class HtmlDocumentExtensions
     {
         public static string GetError(this IHtmlDocument document, string input) => 
             document?.QuerySelector($"span[id='{input}-error']")?.TextContent;
+            
+        public static void AssertErrorMessage(this IHtmlDocument resultDocument, string inputName, string expectedMessage)
+        {
+            var expected = HtmlDocumentHelpers.FullErrorMessage(expectedMessage);
+            var actual = resultDocument.GetError(inputName);
+            Assert.Equal(expected, actual);
+        }

```

Comment:
Would the ideal be that we use this extensively and strip out FullErrorMessage from TestRunnerBase?

Reply:
If you guys think its useful, then yeah, keen to get it adopted wider